SAMtags.tex

```diff
@@ -58,6 +58,7 @@ \section{Standard tags}
   {\tt BC} & Z & Barcode sequence \\
   {\tt BQ} & Z & Offset to base alignment quality (BAQ) \\
   {\tt CC} & Z & Reference name of the next hit \\
+  {\tt CG} & B,I & BAM-only tag to store the real {\sf CIGAR} if it contains $>$65535 operations\\
```

Comment:
We intend not to use the real CIGAR *in bam only* if we have more than 64k, therefore the phrase "intended to store the real CIGAR..." is misleading.  We have no explicit intentions of deliberately using this for all formats.

However when all said and done we need to decide what we wish to do when we see a CG tag in SAM or in CRAM.  Do we ignore it and honour the real CIGAR, do we complain (it's an indication of using an out-dated piece of software that didn't correctly translate back), or do we try to patch the situation up and migrate CG to the cigar field once more?

My gut feeling is ignoring it is easiest and cleanest, but I could see a reason why issuing a warning may be useful.